README.md

```diff
@@ -71,6 +71,12 @@ bundle install
 bundle exec fastlane unit_tests
 ```
 
+If you would like to get the code coverage for all of the modules within the `iOS-SDK`, run the following:
+```
+bundle install
+bundle exec fastlane coverage
```

Comment:
nit: I tried running without `bundle exec` and looks like the outputs are the same.
For simplicity I think maybe `fastlane coverage` will do? :smile:

Reply:
Using `bundle exec` will ensure that we're all using the same version of Fastlane. If you don't use bundle exec, it will use whatever version you have on your machine.

Reply:
Ohhhh I understand now!!!!
Thanks Sammy!! :smile:

Reply:
Generally thinking i think that just using `fastlane coverage` is totally fine - the reason to use `bundle exec` is to ensure that the right gems are used when running the action. [this SO reply](https://stackoverflow.com/a/6588708) does a better job of explaining it than i can